src/oscar/apps/dashboard/offers/forms.py

```diff
@@ -189,3 +189,11 @@ def save(self, *args, **kwargs):
 class OfferSearchForm(forms.Form):
     name = forms.CharField(required=False, label=_("Offer name"))
     is_active = forms.BooleanField(required=False, label=_("Is active?"))
+    offer_type = forms.ChoiceField(required=False, choices=(("", "---------"),) + ConditionalOffer.TYPE_CHOICES)
+
+    @property
+    def is_voucher_offer_type(self):
+        if self.is_bound and (self.cleaned_data['offer_type'] == ConditionalOffer.VOUCHER):
```

Comment:
I think this can be expressed more succinctly as just

```
return self.is_bound and self.cleaned_data['offer_type'] == ConditionalOffer.VOUCHER
```